integration/openshift/README.md

```diff
@@ -0,0 +1,27 @@
+# Openshift integration with Clear Containers
+
+In this section you can find scripts to setup an Openshift enviornment
```

Comment:
OpenShift should always have a capitalized S.
Could replace "In this section you can find" with "This section contains"